src/pages/CallBack.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+
+import { TP } from '../components/THeadings';
+import TProgressCircle from '../components/TProgressCircle';
+import { SdkErrors, throwError, UserApps } from 'tonomy-id-sdk';
+import './main.css';
+
+export default function CallBackPage() {
+    useEffect(() => {
+        const params = new URLSearchParams(window.location.search);
+        const requests = params.get('requests');
+
+        if (!requests) throwError("requests parameter doesn't exists", SdkErrors.MissingParams);
+        const username = params.get('username');
+        if (!username) throwError("username parameter doesn't exists", SdkErrors.MissingParams);
+        const accountName = params.get('accountName');
+        if (!accountName) throwError("accountName parameter doesn't exists", SdkErrors.MissingParams);
+        verifyRequests(requests, username, accountName);
+    }, []);
+
+    async function verifyRequests(requests: string, username: string, accountName: string) {
+        const result = await UserApps.verifyRequests(requests);
+
+        const redirectJwt = result.find((jwtVerified) => jwtVerified.payload.origin !== location.origin);
+        const ssoJwt = result.find((jwtVerified) => jwtVerified.payload.origin === location.origin);
+        if (!redirectJwt) {
+            throwError('missing jwt', SdkErrors.JwtNotValid);
```

Comment:
This is not a JwtNotValid error type.
1. Only the Sdk can throw a SdkErrors type. If we throw an error here it should be of a new type. I called this ApplicationError in ID, so maybe here we can call SSOApplicationError...
2. the problem is not with the JWT, but with the requests

Because you are in the UI component, you do not need to have a error type because you can immediately handle it and show ( or not show) a display to the user